.idea/libraries/jetty.xml

```diff
@@ -1,7 +1,7 @@
 <component name="libraryTable">
   <library name="jetty">
     <CLASSES>
-      <root url="jar://$PROJECT_DIR$/build/third_party/java/jetty/jetty-repacked.jar!/" />
+      <root url="jar://$PROJECT_DIR$/third_party/java/jetty/jetty-repacked.jar!/" />
```

Comment:
> this file also doesn't exist until you build it...

Do you mean `third_party/java/jetty/jetty-repacked.jar`? It does exist actually:

```
$ git co master && ./go clean && find . -name jetty-repacked.jar
(in /Users/alb/workspace/selenium)
./third_party/java/jetty/jetty-repacked.jar
```